### PR TITLE
Added additional configuration options

### PR DESCRIPTION
**Pull request by pmccarren**
- table_definition_cache
- sysdate_is_now
- query_cache_type
- log_queries_not_using_indexes
- binlog_rows_query_log_events
- gtid_mode
- enforce_gtid_consistency
- binlog_checksum
- sync_master_info
- sync_relay_log
- sync_relay_log_info
- master_verify_checksum
- slave_net_timeout
- slave_sql_verify_checksum

### CODE REVIEW

File: templates/default/my.cnf.cluster.erb [hunk 1]
```diff
@@ -55,6 +55,10 @@ slave_load_tmpdir = <%= node["percona"]["server"]["tmpdir"] %>
 character_set_server = <%= node["percona"]["server"]["character_set"] %>
 collation_server  = <%= node["percona"]["server"]["collation"] %>
 
+<% unless node["percona"]["server"]["report_host"].enpty? %>
```
Comment (jharley) [inline templates/default/my.cnf.cluster.erb]: Minor typo which will blow up ERB generation: 'enpty' -> 'empty'

Comment (pmccarren) [inline templates/default/my.cnf.cluster.erb]: @jharley thanks! Just pushed an patch

**Comment by phlipper** [comment]
@pmccarren thank you very much for this! I will be happy to merge this, but first, please rebase this on top of the latest master, revert the version bump, and update the README with the new attributes. Thanks!

**Comment by devsibwarra** [comment]
Sorry about the rebase mess.  My git-fu is lacking

**Comment by devsibwarra** [comment]
@phlipper Picking up for @pmccarren since he has moved on to bigger and better things.  I've re-based to the current master (w/ major assist from @toejough), removed the version bump, and updated README.

Let me know if there are other changes or issues discovered, since it seems like the Travis checks aren't quite right

**Comment by damacus** [comment]
hey @phlipper ans @devsibwarra can someone rebase this and I'll see if I can get in a new release.

@devsibwarra are you interested in helping maintain this cookbook going forward? We're always looking for new maintainers

**Comment by damacus** [comment]
## Closing due to inactivity. 

If this is still an issue please reopen or open another PR. 

Alternatively drop by the #sous-chefs channel on the [Chef Community Slack](http://community-slack.chef.io/) and we'll be happy to help! 

Thanks,
Sous-Chefs

**Pull request closed without merging**